libs/platform/src/lib/components/table/table.component.scss

```diff
@@ -378,6 +378,10 @@
         padding: 1rem;
         text-align: center;
     }
+
+    .fd-dnd-item {
+        position: inherit;
```

Comment:
`inherit`[ will not work on IE](https://caniuse.com/?search=inherit)

Reply:
It will. The first block on the caniuse about other inherit, please consider the second block.

![image](https://user-images.githubusercontent.com/20265336/121009330-b460cb00-c79c-11eb-9596-aa3f6dd43d3e.png)

Reply:
Oh yeah. Okay, sorry about this 😄